aws/sagemaker_studio/README.md

```diff
@@ -0,0 +1,45 @@
+# Instructions for Running RAPIDS + SageMaker Studio + HPO
```

Comment:
```suggestion
# Instructions for Running RAPIDS + SageMaker Studio
```
Keeping this README high level since the estimator can be used for other workflows